File: C#/baseball/Answer.cs
using System;
using System.Linq;

namespace BaseballCSharp
{
    public class Answer : NumberContainer
    {
        public void Create()
        {
            Random random = new Random();

            while (true)
            {
                for (int i = 0; i < _numbers.Length; i++)
                    _numbers[i] = random.Next(Constant.MaxNumber);

                if (_numbers.ToHashSet().Count == Constant.Digit)
                    break;
            }
        }

        protected override string GetPrefix()
        {
            return "[정답] ";
        }
    }
}

File: IoT기반_스마트팩토리_SW개발_전문가_과정/C,C++/baseball/baseball_10/Answer.cpp
#include <iostream>
#include "Constant.h"
#include "Answer.h"
using namespace std;

void Answer::create()
{
    srand(0);

    while (true)
    {
        for (int i = 0; i < DIGIT; i++)
            numbers[i] = rand() % MAX_NUMBER;

        if (numbers[0] != numbers[1] && numbers[1] != numbers[2] && numbers[2] != numbers[0])
            break;
    }
}

void Answer::print()
{
    cout << "[정답] ";
    for (int i = 0; i < DIGIT; i++)
        cout << numbers[i] << " ";
    cout << endl;
}

int Answer::at(int index)
{
    return numbers[index];
}
